MeetupPlatformApi/Persistence/Migrations/20220402152002_AddLocationToMeetup.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class AddLocationToMeetup : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
```

Comment:
I would never believe that the `ApplicationContextModelSnapshot` was not affected by that migration. Why didn't You include it to the PR?